src/app/wallet/calculator/calculator.component.css

```diff
@@ -7,19 +7,13 @@ h3 {
   text-align: center;
 }
 
-input, select {
-  display: block;
-  margin: 5px auto;
-  border-radius: 2px;
-  border: 1px solid #7A7A7A;
-  min-width: 15rem;
-  min-height: 1.6rem;
-  background-color: #FFF;
+.form-group {
+  text-align: center;
 }
 
-select {
-  height: 2rem;
-  width: 15.5rem;
+mat-form-field {
```

Comment:
Will update the class naming convention for each mat-form-field wrapped inputs. I accept this as a parent styling for every input for now

Reply:
I think with the view encapsulation provided by Angular for each component, the style won't affect other mat-form-fields.

Reply:
Yeah @danjodanjo. As @victorhuberta mentioned, only the calculator component will have those padding because of angular css encapsulation.

Reply:
I also think it'll be a good idea if we setup sass instead of using regular css as the project grows